tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "allowJs": true,
+    "checkJs": false,
```

Comment:
What happens if we enable this flag?

Reply:
From http://www.typescriptlang.org/docs/handbook/compiler-options.html:
"Report errors in .js files. Use in conjunction with --allowJs."